tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -551,6 +563,232 @@ describe(__filename, () => {
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_FIREFOX,
+        name: 'Alerts & Update',
+        slug: 'alert-update',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_FIREFOX,
+        name: 'Security',
+        slug: 'security',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_FIREFOX,
+        name: 'Anime',
+        slug: 'anime',
+        type: ADDON_TYPE_STATIC_THEME,
+      },
+    ];
+
+    const addon = createInternalAddonWithLang({
+      ...fakeAddon,
+      categories: {
+        [CLIENT_APP_FIREFOX]: [],
+      },
+      type: ADDON_TYPE_EXTENSION,
+    });
+
+    dispatchClientMetadata({ clientApp: CLIENT_APP_FIREFOX, store });
+    store.dispatch(loadCategories({ results }));
+
+    const root = render({
+      addon,
+      store,
+    });
+
+    expect(
+      root.find('.AddonMoreInfo-related-categories').find(Link),
+    ).toHaveLength(0);
+  });
+
+  it('does not render related categories when categories for add-on do not exist', () => {
+    const results = [
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_ANDROID,
+        name: 'Alerts & Update',
+        slug: 'alert-update',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_ANDROID,
+        name: 'Blogging',
+        slug: 'blogging',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_ANDROID,
+        name: 'Games',
+        slug: 'Games',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_ANDROID,
+        name: 'Security',
+        slug: 'security',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_FIREFOX,
+        name: 'Anime',
+        slug: 'anime',
+        type: ADDON_TYPE_STATIC_THEME,
+      },
+    ];
+
+    const addon = createInternalAddonWithLang({
+      ...fakeAddon,
+      categories: {
+        [CLIENT_APP_FIREFOX]: ['alert-update', 'security'],
+      },
+      type: ADDON_TYPE_EXTENSION,
+    });
+
+    dispatchClientMetadata({ clientApp: CLIENT_APP_FIREFOX, store });
+    store.dispatch(loadCategories({ results }));
+
+    const root = render({
+      addon,
+      store,
+    });
+
+    expect(
+      root.find('.AddonMoreInfo-related-categories').find(Link),
+    ).toHaveLength(0);
+  });
+
+  it('renders errors when API error occurs', () => {
+    const errorHandler = new ErrorHandler({
+      id: 'some-error-handler-id',
+      dispatch: store.dispatch,
+    });
+
+    errorHandler.handle(
+      createApiError({
+        response: { status: 404 },
+        apiURL: 'https://some/api/endpoint',
+        jsonResponse: { message: 'not found' },
+      }),
+    );
+
+    const root = render({ errorHandler, store });
```

Comment:
```suggestion
    const root = render({ errorHandler });
    
```

I don't think you need to pass the `store`. It should be passed automagically.

nit: blank line after `render()`